### ISSUE TITLE
New setting: "Exclude single digit tab markers"

### ISSUE THREAD

**Issue by JWaldenback** [david-tejada/rango#185]
![image](https://github.com/david-tejada/rango-talon/assets/19432538/1c5b30d3-74ef-4582-9c3c-4d10fee25af5)

**Comment by david-tejada** [comment]
Ok, I thought you wanted digits as tab markers instead of letters. What you want is an option for the first 8 tabs to not have tab markers because for those you can use `go tab <number>`. Is that right?

**Comment by JWaldenback** [comment]
Oh, sorry I realized now I've used the wrong terminology. What I meant was "Exclude single letter markers" but now that you ask I think the implementation you talk about could be even better because Rango will not interfere with the talon community's default functionality then. I suppose Rango could even drop the go in `go tab` and just make it `tab` command if this is implemented, right? Or is it risky to use the same voice command?

**Comment by JWaldenback** [comment]
If it is risky I would be very content with just having a setting to exclude the single letter hints and only show double letter hints :-)

**Comment by david-tejada** [comment]
Ok, what you want is that tab markers start at "AA", right? What is the reason for this? With hints it makes sense because of direct clicking conflicting with pressing keys. But I fail to see what's the advantage here for tabs.

I don't think Rango interferes with the community implementation since one takes numbers and the other letters. In the beginning I wanted to use `tab <letters>`. The problem I had with this is I got too many misrecognitions with instances of, say, "tab air" being recognized as `cap air` and similar. If it works to you you can change the command so you only have to say `tab <letters>`.

**Comment by JWaldenback** [comment]
It's not important what letters the tab markers start with and they don't need to be in order. Just that the markers are two letters instead of one letter.

Mainly it's because I could change the Rango command to `tab <letters>` then without it interfering with the community implementation. So I will be able to use `tab one` and `tab air air` interchangeably. And then the second reason is that I am more used to having markers double letters as I've excluded single letters from the regular markers.

**Comment by JWaldenback** [comment]
This is not a biggie so if it's cumbersome to implement please don't hesitate to close this issue. I just find it reasonable to be able to use double tab markers on all places, it kind of keeps everything consistent :-)

**Comment by david-tejada** [comment]
It's not that it's cumbersome but if I'm going to implement something I need to know what's the utility.

The community implementation is actually `go tab <number>`. Have you tried changing both commands to use `tab`? Because I used that prior to changing to `slot` and I got no conflicts whatsoever with the community implementation. The only reason I changed is because `tab` would sometimes get interpreted as `cap`. Give it a try, leave the community command as `tab <number>`  if that's what you have now and change the rango command to `tab <user.rango_target>` and let me know if you have any conflicts between both.
